be/src/storage/storage_engine.cpp

```diff
@@ -915,19 +915,41 @@ Status StorageEngine::_do_sweep(const std::string& scan_root, const time_t& loca
     return res;
 }
 
-void StorageEngine::start_delete_unused_rowset() {
-    std::lock_guard lock(_gc_mutex);
-    for (auto it = _unused_rowsets.begin(); it != _unused_rowsets.end();) {
-        if (it->second.use_count() != 1) {
-            ++it;
-        } else if (it->second->need_delete_file()) {
-            VLOG(3) << "start to remove rowset:" << it->second->rowset_id()
-                    << ", version:" << it->second->version().first << "-" << it->second->version().second;
-            Status status = it->second->remove();
-            VLOG(3) << "remove rowset:" << it->second->rowset_id() << " finished. status:" << status;
-            it = _unused_rowsets.erase(it);
+void StorageEngine::delete_unused_rowset(int32_t* deleted_pct) {
```

Comment:
why not return double?
```suggestion
double StorageEngine::delete_unused_rowset() {
```